app/src/main/java/com/infomaniak/drive/data/api/ApiRepository.kt

```diff
@@ -257,6 +262,10 @@ object ApiRepository {
         return callApi(ApiRoutes.shareLink(file), PUT, body)
     }
 
+    fun getCategory(driveId: Int): ApiResponse<Array<Category>> {
+        return callApi(ApiRoutes.createCategory(driveId), GET)
+    }
```

Comment:
`getCategory`, but the Api call is to `createCategory`?

Reply:
Oh, it's a `GET` call. We don't have this, do we??

Reply:
Yes I added it for the tests to be able to assert changes or deletion after calls that don't return the changes